components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandler.java

```diff
@@ -574,4 +574,20 @@ private static void addUserAttributesToCache(AccessTokenDO accessTokenBean,
             AuthorizationGrantCache.getInstance().addToCacheByToken(authorizationGrantCacheKey, grantCacheEntry);
         }
     }
+
+    private boolean getRenewRefreshTokenProperty(String renewRefreshToken) {
```

Comment:
we need to change the method name to "isRenewRefreshToken(..)" and use it like
if(! isRenewRefreshToken(..) {
..
} else {
}

As it is more readable

Reply:
fixed with 0d23b1b5c857ce0897ca3526455f2de3fc55784e